packages/@okta/vuepress-site/docs/concepts/third-party-cookies/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cross-Site Cookie Blocking
+---
+
+# Cross-Site Cookie Blocking
+
+Cross-site cookie blocking is a privacy-protection feature that is being added to web browsers like Safari and Firefox. It can disrupt certain customer-hosted flows in Okta.
```

Comment:
we should add a comment that we expect the practice of restricting third-party cookies to become common across the browser ecosystem.